### PR TITLE
feat(linux) Add `tzdata` package

### PR DESCRIPTION
**Pull request by gounthar**
Signed-off-by: gounthar <gounthar@gmail.com>

I'm sorry for the indentation of [this file](https://github.com/jenkinsci/docker-agent/pull/337/files#diff-3f163a09ffba37e91b3af585fae100b7332047433e42b848836c4a492d10b6d7), I have changed my `CRLF` git config though.

This is a work in progress, please don't review it now.

I have just added `tzdata` and added a little bit of cleanup.
Even without installing `tzdata`, the machine is already configured as `Etc/UTC`:

```bash
cat /etc/timezone
Etc/UTC
```
I [test](https://github.com/jenkinsci/docker-agent/pull/337/files#diff-1059443e6c79adcf296eee8ff7fd5b3fade119e5971104d18119cce68068f959R102) several utilities brought by tzdata (in fact, I test their presence and the fact we can launch them, I don't attempt any fancy command line magic).
I also test that we are using the `Etc/UTC` timezone by default.

- [X] Make sure you are opening from a **topic/feature/bugfix branch** (right side) and not your main branch!
- [X] Ensure that the pull request title represents the desired changelog entry
- [X] Please describe what you did
- [X] Link to relevant issues in GitHub or Jira
- ~~Link to relevant pull requests, esp. upstream and downstream changes~~
- [X] Ensure you have provided tests - that demonstrates feature works or fixes the issue

### CODE REVIEW

**Comment by gounthar** [comment]
We're still at 35 tests, my [test](https://github.com/jenkinsci/docker-agent/pull/337/files#diff-1059443e6c79adcf296eee8ff7fd5b3fade119e5971104d18119cce68068f959R102) is not played. 🤔 
I must have forgotten something. 😊

File: tests/tests.bats [hunk 1]
```diff
@@ -99,6 +99,33 @@ ARCH=${ARCH:-x86_64}
   local TEST_AGENT_WORKDIR="/home/test-user/something"
   local sut_image="${SUT_IMAGE}-tests-${BATS_TEST_NUMBER}"
 
+@test "[${SUT_IMAGE}] `tzdata` is correctly installed" {
```
Comment (dduportal) [inline tests/tests.bats:102]: this test seems to be specified *inside* another test already. You might want to move it

Comment (gounthar) [reply tests/tests.bats:103]: 🤦‍♂️ 
I had not seen that, thanks for the hint. 😊

File: tests/tests.bats [hunk 2]
```diff
@@ -150,3 +150,22 @@ docker buildx bake \
...
+  run docker exec "${cid}" zdump --version
+  assert_success
+
+  run docker exec "${cid}" sh -c "command -v zic"
+  assert_success
+  run docker exec "${cid}" zic --version
+  assert_success
+
+  cleanup "$cid"
+}
```
Comment (dduportal) [inline tests/tests.bats:171]:
```suggestion
}

```

Missing endline

Comment (gounthar) [reply tests/tests.bats:171]: As always 😊

**Review by dduportal — APPROVED** [review]

**Pull request merged**